tests/acceptance/EditSignUpConfirmationEmailCest.php

```diff
@@ -7,34 +7,44 @@
 use MailPoet\Test\DataFactories\Settings;
 
 class EditSignUpConfirmationEmailCest {
-  public function edit(\AcceptanceTester $i) {
+  public function editSignUpConfContentAndVerify(\AcceptanceTester $i) {
     $i->wantTo('Edit sign up confirmation email');
-
     // make sure sign up confirmation is enabled
     $settings = new Settings();
     $settings->withSender('Confirmation Test From', 'from-confirmation-test@example.com');
     $settings->withConfirmationEmailEnabled();
     $forms = new Form();
     $forms->withDefaultSuccessMessage();
+    $confirmationEmailSubject = 'Confirmation email subject';
 
     $i->login();
     $i->amOnMailPoetPage('Settings');
     $i->click('[data-automation-id="signup_settings_tab"]');
     $i->waitForText('Enable sign-up confirmation');
-
     // edit confirmation email
-    $i->fillField('[data-automation-id="signup_confirmation_email_subject"]', 'Confirmation email subject');
+    $i->fillField('[data-automation-id="signup_confirmation_email_subject"]', $confirmationEmailSubject);
     $i->fillField('[data-automation-id="signup_confirmation_email_body"]', 'Confirmation email body [activation_link]link[/activation_link]');
-
     $i->click('[data-automation-id="settings-submit-button"]');
-
+    // create form and subscribe
     $i->createFormAndSubscribe();
-
+    // performing some clicks and getting back to verify the content
```

Comment:
I'll add the explanation from the task as a comment, it won't be lost over time this way.